acceptance/chaos_test.go

```diff
@@ -56,20 +64,55 @@ func TestChaos(t *testing.T) {
 	var stalled int32
 	clients := make([]struct {
 		sync.RWMutex
-		db      *client.DB
+		db      *sql.DB
 		stopper *stop.Stopper
 		count   int64
 	}, num)
 
 	// initClient requires that the caller holds the client's write lock.
 	initClient := func(i int) {
-		db, dbStopper := makeClient(t, c.ConnString(i))
+		db := makePGClient(t, c.PGUrl(i))
```

Comment:
Add a comment that each client is talking to a different node. I almost pointed out that you could share an `sql.DB` across all clients before realizing this was being done.

Oh, and you can assign to `clients[i].db` directly here. Not much point in the `db` variable.